### ISSUE TITLE
Elementum shows results only from one tracker even though I've selected many

### ISSUE THREAD

**Issue by georgekech** [elgatito/plugin.video.elementum#862]
It's my first post of this page so I'm not sure if I'm doing it right. Elementum sometimes it needs you to keep pressing the same result to show you the available torrents and when it does after many "no link", for material in English, it's always the RARBG tracker. 

I'm not sure how to paste my log but here it is: 
[kodi.log](https://github.com/elgatito/plugin.video.elementum/files/7850394/kodi.log)

**Comment by antonsoroko** [comment]
in your log i can see messages like
`1337x returned  0 results in 1.0 seconds`
although query for 1337x work for me. copy queries from your log file (search in file for `1337x search URL:`, for example) and open in browser on same device - maybe your ISP block them or you hit cloudflare defense or similar - is so - try VPN or similar tech. or maybe you use VPN that blocked by torrent sites (or simply rate limit defense on trackers is reached for IP address you use).

about rarbg - it definitely has rate limiting, so if you search too often (with small delay) - it will give 0 results.

---

btw, you can use http://elementum.surge.sh/contribute/log/ for convenience - as stated in template when you create issue. also - it is better to use template (so you will provide all necessary and not "free-style" text.

**Comment by georgekech** [comment]
You're right. Some of those trackers are block by my ISP. Thanks. 

There's another issue which I cannot figure out. In some 4k video streams the image is alternate by the player. They look like negatives. Take a look and tell me which setting I'm missing.
![IMG_2230](https://user-images.githubusercontent.com/63591129/149083796-80f27239-2204-423d-8bcf-21e2ce0e549d.JPG)

**Comment by dlxmax** [comment]
Looks like you're playing a Dolby Vision ("DV") video stream on a non-Dolby Vision tv.

**Comment by antonsoroko** [comment]
also this is not elementum related question :-D
elementum does not play anything - Kodi does.

so either your kodi/TV configured incorrectly or you play something unsupported (like HDR - looks like this "Dolby Vision" is also HDR).

**Comment by elgatito** [comment]
Closing the issue as it is seems to be related to provider search.
